dbt_copilot_helper/commands/copilot.py

```diff
@@ -25,15 +26,36 @@
 
 WAF_ACL_ARN_KEY = "waf-acl-arn"
 
+SERVICE_TYPES = [
+    "Load Balanced Web Service",
+    "Backend Service",
+    "Request-Driven Web Service",
+    "Static Site",
+    "Worker Service",
+]
+
 
 def list_copilot_local_environments():
     return [
         path.parent.parts[-1] for path in Path("./copilot/environments/").glob("*/manifest.yml")
     ]
 
 
+def is_service(path: PosixPath) -> bool:
+    with open(path) as manifest_file:
+        data = yaml.safe_load(manifest_file)
+        if not data:
+            return False
```

Comment:
Somethings gone wrong rather than it's just not a service. Feels like it should exit the command with an error to me.